tests/unit/helpers/test_db_workspace_utils.py

```diff
@@ -0,0 +1,37 @@
+# pylint: disable=protected-access
```

Comment:
DO NOT DISABLE PYLINT

Reply:
Yes private method UT will happen by calling the parent method, no need to test private methods directly.

Reply:
Done